traveler-ios-ui/TravelerKitUI/ViewControllers/OrderDetailViewController.swift

```diff
@@ -106,7 +106,12 @@ class OrderDetailViewController: UITableViewController {
     }
 
     @IBAction func didRequestTickets(_ sender: Any) {
+        guard let order = order else {
+            Log("Missing order", data:  nil, level: .error)
+            return
+        }
 
+        Traveler.emailTickets(order: order, delegate: self)
```

Comment:
display a HUD?...user has no idea their request is being processed.

I personally dont think a HUD is necessary because I dont think it is required to block the UI. I couldn't think of a better solution, you can potentially bring this up with UX and get their input. But some sort of indicator is necessary. and it should be simple and not require a lot of work on your part. I think we would be happy with a HUD at the end of the day